rhoas-cli-service-registry/README.adoc

```diff
@@ -120,23 +124,27 @@ You are prompted to enter the `Name` for the {registry} instance.
 ====
 --
 
-. Enter the following command to verify that the {registry} instance is ready to use.
+. Verify that the {registry} instance is ready to use.
 +
 --
 This command shows that the {registry} instance is ready to use,
```

Comment:
Line 126-127/130-131: Move to Line 149? (after the command output).

Reply:
Done